src/com/sap/piper/MapUtils.groovy

```diff
@@ -0,0 +1,10 @@
+package com.sap.piper
+
+import com.cloudbees.groovy.cps.NonCPS
+
+class MapUtils implements Serializable {
+    @NonCPS
+    static isMap(object){
```

Comment:
It will be reused [here](https://github.com/SAP/jenkins-library/pull/58/files#diff-e898b5375d7d283aa9c425eac98ce8d2R189) and [here](https://github.com/SAP/jenkins-library/pull/60/files#diff-18e45a2c716552b4acd481e2c8952d34R208).

Reply:
&lt;nitPicking&gt;Aahhh ... But pull requests should be independent of each other. :-) &lt;/nitPicking&gt;